### ISSUE TITLE
PvArgumentError when running the File Demo command using Node JS

### ISSUE THREAD

**Issue by Javeed-JargonHandlers** [Picovoice/rhino#100]
Hi, 

NodeJS: v12.13.0 
I am using Node JS global packages. I have installed : **npm install -g @picovoice/rhino-node-demo**

When I am running the command to transciber --> rhn-file-demo --context_path resources/contexts/linux/coffee_maker_mac.rhn --input_audio_file_path resources/audio_samples/test_within_context.wav

I am getting the following error.

 throw new PvArgumentError(
    ^

ReferenceError: PvArgumentError is not defined
    at fileDemo (/home/jav/.nvm/versions/node/v12.13.0/lib/node_modules/@picovoice/rhino-node-demo/file.js:63:5)
    at Object.<anonymous> (/home/jav/.nvm/versions/node/v12.13.0/lib/node_modules/@picovoice/rhino-node-demo/file.js:132:1)

Thanks in advance

**Comment by dbartle** [comment]
I'm assuming you're running it on a mac. It looks like the path to the context file has a typo in it:

```
--context_path resources/contexts/linux/coffee_maker_mac.rhn
```

should be:

```
--context_path resources/contexts/mac/coffee_maker_mac.rhn
```

The demo should obviously produce a better error message, (seems like it throws an exception while trying to throw an exception unfortunately), so I will keep this issue open until that's improved.

**Comment by dbartle** [comment]
The bug preventing the proper error message was fixed in 37a33bc38383fc0446accdc8209edd63253f07b8
